src/index.js

```diff
@@ -6,5 +6,60 @@ import './css/base.scss';
 
 // An example of how you tell webpack to use an image (also need to link to it in the index.html)
 import './images/turing-logo.png'
+import Customer from "./Customer"
 
-console.log('This is the JavaScript entry file - your code begins here.');
+const roomNumber = document.getElementById('roomNumber')
+const dateStayed = document.getElementById('dateStayed')
+const totalCost = document.getElementById('totalSpent')
+const date = document.querySelector('.date')
+const roomType = document.querySelector('.roomType')
+
+const allCustomersData =
+  fetch("http://localhost:3001/api/v1/customers")
+    .then(response => response.json())
+
+const allRoomsData =
+  fetch("http://localhost:3001/api/v1/rooms")
+    .then(response => response.json())
+
+const allBookingsData =
+  fetch("http://localhost:3001/api/v1/bookings")
+    .then(response => response.json())
```

Comment:
May want to move all api requests to a separate file for separation of responsibilities since there will be so many in this project. Also, you may be able to make these one dynamic function expression and pass in the last bit of the URL (customers, rooms, or bookings) since they are otherwise the same.